config/app.container-definition.json

```
@@ -107,7 +111,7 @@
     },
     {
       "name": "STORAGE_BACKEND",
-      "value": "s3"
+      "value": "cdn"
```

I just realized we want to pass this in for each env (which also means adding `STORAGE_BACKEND` to each file:
```suggestion
      "value": "{{ .STORAGE_BACKEND }}"
```